prow/gerrit/adapter/adapter_test.go

```diff
@@ -704,18 +763,18 @@ func TestProcessChange(t *testing.T) {
 			},
 		}
 
-		testInstance := "https://gerrit"
 		fakeProwJobClient := prowfake.NewSimpleClientset()
 		fakeLastSync := client.LastSyncState{testInstance: map[string]time.Time{}}
 
 		for _, tc := range testcases {
 			fakeLastSync[testInstance][tc.change.Project] = timeNow.Add(-time.Minute)
 		}
 
+		gc := &fgc{}
```

Comment:
Similar to above I would consider `var gc fgc` (and `gc: &gc` below) to be better practice as we never set gc to nil so there's no need to have to deal with pointers.

Reply:
FYI I think you also need to call [`t.Parallel()`](https://golang.org/pkg/testing/#T.Parallel) at the beginning of the anonymous run function if you want the test cases to run in parallel. ref: https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks

Reply:
Thank you for the review. Fixed.